_std/defines/input.dm

```diff
@@ -1,19 +1,21 @@
 // haha hope you never need more than 16 of these
-#define KEY_FORWARD   0x0001
-#define KEY_BACKWARD  0x0002
-#define KEY_LEFT      0x0004
-#define KEY_RIGHT     0x0008
-#define KEY_RUN       0x0010
-#define KEY_THROW     0x0020
-#define KEY_POINT     0x0040
-#define KEY_EXAMINE   0x0080
-#define KEY_PULL      0x0100
-#define KEY_OPEN      0x0200
-#define KEY_BOLT      0x0400
-#define KEY_SHOCK     0x0800
+// this used to use hexadecimal, now we can have 32, fun
```

Comment:
that's not how bitflags work.
we're limited to 24.

Reply:
My mistake, thought it was 32